### ISSUE TITLE
 delete "playground" from go.work first

### ISSUE THREAD

**Issue by gedw99** [mearaj/bhagad-house-booking#26]
front-build-mac does not work unless i delete "playground from go.work

**Comment by mearaj** [comment]
Yeah, actually go.work should be hidden and depends upon local setup. I didn't remove it because it was still useful in Docker setup. 
Will add playground folder it to and update it.
Sorry for the confusion.
